ansible/roles/ci/deploy/upload_debs/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Defaults aptly url
+
+aptly_url: http://10.10.10.62:8080
+repository_name: shadow-private
+gpg_password: shadow
```

Comment:
yes there are options to have the passphrase or link to the key files directly on the server running aptly. I tried it quickly and couldn't get them to work, but it should be possible.